### PR TITLE
Replace unidecode with text-unidecode.

### PR DESCRIPTION
**Pull request by jonatankawalek**
This PR addresses "Unidecode has a problematic OSS license #1508".

### CODE REVIEW

**Comment by felixdittrich92** [comment]
Hi @jonatankawalek :wave:,

Thanks for the PR i agree with the license mismatch, but `text-unidecode` seems not to get any updates anymore. Maybe https://github.com/anyascii/anyascii is a better alternative wdyt ?

**Comment by jonatankawalek** [comment]
Hi @felixdittrich92,

You're right - text-unidecode seems stale. Anyascii looks to be a better choice. Let me replace it with that.

**Comment by felixdittrich92** [comment]
@jonatankawalek 
Please a `anyascii.*` to 
pyprojet.toml L147
```
[[tool.mypy.overrides]]
module = [
```
And run `make style` afterwards to fix formating issues :)
You can use `make quality` to check that all typings are correct from your code changes (If there is listed something you haven't changed ignore it)

**Comment by jonatankawalek** [comment]
@felixdittrich92 

Ran make style and fixed flags for my code chages (there were other style issues too, but not to do with my changes).
Make quality doesn't flag anythin for my code changes

File: pyproject.toml [hunk 1]
```diff
@@ -145,6 +145,7 @@ implicit_reexport = false
 
 [[tool.mypy.overrides]]
 module = [
+    "anyascii.*",
```
Comment (felixdittrich92) [inline pyproject.toml:148]: indentation :)

Comment (jonatankawalek) [reply pyproject.toml]: ahh, tabs vs spaces - fixed

**Review by felixdittrich92 — APPROVED** [review]
Thanks for the PR and replacement :+1:

**Pull request merged**